syntax = "proto3";

package pets;

service Pets {
  rpc GetPets (PetsRequest) returns (PetsReply) {}
}

message PetsRequest {
  int32 user_id = 1;
  int32 pet_id = 2;
}

message PetsReply {
  string pets_message = 1;
  string error = 2;
  repeated string pets = 3;
  string pet_name = 4;
}
